validation/src/test/scala/hmda/validation/rules/lar/quality/Q059Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q059Spec extends LarEditCheckSpec with BadValueUtils {
+  property("Lars with loan type equal to 1 must pass") {
```

Comment:
In the validation tests, we tended to do the equivalent of "loan type other than 2, 3, or 4" here.  Is there a conscious decision to do it differently for the quality edits, figuring that they will already have passed validation? (I am OK with that, just want to make sure it's conscious. And apologies if I've missed it happening elsewhere; there have been a lot of GitHub notifications flying by, which is great! but may mean I missed some, particularly for things that are closed by the time I see them.)